### PR TITLE
deCONZ 2.5.87 (stable)

### PR DESCRIPTION
**Pull request by pvizeli**

### CODE REVIEW

**Comment by wovka88** [comment]
There is a typo in the Version number, on the server the files are still with deconz-2.05.87-qt5.deb and not 2.5.87.

Can you correct it please?

**Review by fi-sch — COMMENTED** [review]
I think it should be:
"DECONZ_VERSION": "2.05.87"

:)

EDIT: Oops, just noticed the comment above.

**Comment by Knodd** [comment]
v2.5.88 has been released as a hotfix for v2.5.87

File: deconz/build.json [hunk 1]
```diff
@@ -5,6 +5,6 @@
     "armhf": "homeassistant/armhf-base-raspbian:buster"
   },
   "args": {
-    "DECONZ_VERSION": "2.05.84"
+    "DECONZ_VERSION": "2.5.87"
```
Comment (frenck) [inline deconz/build.json:8]:
```suggestion
    "DECONZ_VERSION": "2.5.88"
```

File: deconz/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 6.5.0
+
+- Bump deCONZ to 2.05.87
```
Comment (frenck) [inline deconz/CHANGELOG.md:5]:
```suggestion
- Bump deCONZ to 2.05.88
```

**Review by frenck — CHANGES_REQUESTED** [review]
They have done a hotfix 2 hours ago, so 87 has an issue.

At the time of writing this review comment, the deps are not finished building yet (as it seems).

<https://github.com/dresden-elektronik/deconz-rest-plugin/releases/tag/v2.5.88>

File: deconz/build.json [hunk 3]
```diff
@@ -5,6 +5,6 @@
     "armhf": "homeassistant/armhf-base-raspbian:buster"
   },
   "args": {
-    "DECONZ_VERSION": "2.05.84"
+    "DECONZ_VERSION": "2.5.88"
```
Comment (frenck) [inline deconz/build.json:8]:
```suggestion
    "DECONZ_VERSION": "2.05.88"
```

**Review by frenck — APPROVED** [review]

**Pull request merged**